package.json

```
@@ -22,6 +22,7 @@
 		"@ntvr/eslint-config": "^1.0.2",
 		"@ntvr/stylelint-config": "^3.0.0",
 		"@stylistic/eslint-plugin": "^2.0.0",
+		"@tailwindcss/postcss7-compat": "npm:@tailwindcss/postcss7-compat@2.0.1",
```

couldn't this be just 
```suggestion
		"@tailwindcss/postcss7-compat": "2.0.1",
```
?

---

i will rollback this, as it was connected with tailwind autocomplete, with no luck